app/pages/project/components/ProjectNavbar/components/ProjectTitle/ProjectTitle.jsx

```diff
@@ -13,43 +20,74 @@ const H1 = styled.h1`
   text-shadow: 0 ${pxToRem(2)} ${pxToRem(3)} rgba(0,0,0,0.5);
 `;
 
-const StyledLink = styled(IndexLink).attrs({
+export const StyledLink = styled(IndexLink).attrs({
   activeClassName: 'active'
 })`
-  text-decoration: none;
+  border-bottom: ${pxToRem(3)} solid transparent;
   color: white;
+  text-decoration: none;
+  white-space: nowrap;
 
   &:hover,
   &:focus {
     border-bottom: ${pxToRem(3)} solid white;
   }
 `;
 
-const CheckMarkWrapper = styled.span`
-  font-size: 0.4em;
+export const StyledCheckMarkWrapper = styled.span.attrs({
+  'aria-title': zooniverseApprovedTranslation,
+  role: 'img',
+  title: zooniverseApprovedTranslation
+})`
+  font-size: 0.75rem;
+  margin-left: 1rem;
 `;
 
-const CheckMark = styled.i`
-  color: #00979d;
+export const StyledCheckMark = styled.i`
+  color: ${theme('mode', {
+    light: zooTheme.colors.brand.default
+  })};
+  text-shadow: none;
 `;
 
+export const StyledUnderReview = styled.small`
+  align-self: center;
+  color: ${theme('mode', {
+    light: zooTheme.colors.brand.default
+  })};
+  font-size: ${pxToRem(15)};
+  text-transform: uppercase;
+`;
+
+
 function ProjectTitle({ launched, link, title, underReview }) {
   return (
-    <H1>
-      <StyledLink to={`${link}?facelift=true`}>
-        {underReview && <p><em>Under Review</em></p>}
-        {title}{' '}
-        {launched &&
-          <CheckMarkWrapper className="fa-stack">
-            <i className="fa fa-circle fa-stack-2x" />
-            <CheckMark className="fa fa-check fa-stack-1x" />
-          </CheckMarkWrapper>
-          }
-      </StyledLink>
-    </H1>
+    <ThemeProvider theme={{ mode: 'light' }}>
+      <H1>
+        <StyledLink to={`${link}?facelift=true`}>
+          <span>
+            {title}
+            {launched &&
+              <StyledCheckMarkWrapper className="fa-stack">
+                <i className="fa fa-circle fa-stack-2x" />
+                <StyledCheckMark className="fa fa-check fa-stack-1x" />
+              </StyledCheckMarkWrapper>}
+            </span>
+        </StyledLink>
+        {underReview &&
```

Comment:
this might still show "Under Review" if project has gone through beta (`beta_approved: true`) but then has also been `launch_approved: true`.
https://styled-components-theme.pfe-preview.zooniverse.org/projects/markb-panoptes/nfnorgtest-moths?facelift=true